app/views/emailTemplates/confirmation.volt

```diff
@@ -1,24 +1,25 @@
-
-
 <tbody>
-	<tr>
-		<td style="padding:40px 0  0 0;">
-			<p style="color:#000;font-size: 16px;line-height:24px;font-family:'HelveticaNeue','Helvetica Neue',Helvetica,Arial,sans-serif;font-weight:normal;">
+    <tr>
+        <td style="padding:40px 0  0 0;">
```

Comment:
Is inline styling still needed for email, or can we move this to the head?

Reply:
Yes, for backward compatibility.